src/sage/misc/replace_dot_all.py

```diff
@@ -385,7 +385,11 @@ def make_replacements_in_file(location, package_regex=None, verbose=False, outpu
         write_file.write(replaced_content)  # overwriting the old file contents with the new/replaced content
 
 
-def walkdir_replace_dot_all(dir, file_regex=r'.*[.](py|pyx|pxi)$', package_regex=None, verbose=False):
+excluded_file_regex = 'auto-methods|replace_dot_all'
+
```

Comment:
You may want `r'` like below

Reply:
Do we actually need L388? The argument `excluded_file_regex` of `walkdir_replace_dot_all` gets the default value. No?

Reply:
Thanks. I've removed it in 299ae874a6.